client/actions/userActions.js

```diff
@@ -9,6 +9,19 @@ export function login(response, type){
   };
 }
 
+export function setRedirectUrl(url){
```

Comment:
Bringing to your attention, the way we imported all the constants. In the possible future we'll be having lots of constants due to plenty actions. It's good practice if we import only what we need and not everything in the constants file.

Reply:
Also it's good style if we space our named imports for example in L3:

```
import { authenticationFailed, handleError, throwError } from '../utils/errorHandler';
```
There should be a space between the curly braces.